tests/test_monitor.py

```diff
@@ -1153,13 +1153,13 @@ def test_container_multiplex(self,
                 call(10, 0, '                  signal1: 0 m,'),
                 call(11, 0, '                  signal5: 4.0'),
                 call(12, 0, '              )'),
-                call(13, 0, '       0.000  OneToContainThemAll :: multiplexed_message__SELECT_HELLO('),
+                call(13, 0, '       0.000  OneToContainThemAll :: multiplexed_message('),
```

Comment:
I'm not sure if removing the values of the selector signals here is a good idea. maybe it ought to be formatted more nicely, e.g., `OneToContainThemAll :: multiplexed_message :: SELECT_HELLO`?

Reply:
multiple numeric selectors doesn't look good: `Extended :: 0 :: 0 :: 1`.
Maybe we can print selector name in case of multiple numeric values like `Extended :: S0=0 :: S1=0 :: S2=1`... but theres risk it may not fit to screen

Reply:
I agree. let's try and if we come up with something better, we can change it later...